src/main/groovy/org/ysb33r/gradle/gradletest/TestGenerator.groovy

```diff
@@ -128,16 +128,31 @@ class TestGenerator extends DefaultTask {
      * @return A map of consisting of {@code <TestName,PathToTest>}.
      */
     @Input
-    TreeMap<String,File> getTestMap() {
-        TreeMap<String,File> derivedTestNames = [:] as TreeMap
+    TreeMap<String,TestDefinition> getTestMap() {
```

Comment:
This is the meat of the multi-build file logic.  this method here searches the project folder for gradle files and gradle.kts files and loads them into arrays.  those two arrays becomes the tests that will be executed, one test per build file.